src/screens/TopicMemberScreen/index.js

```diff
@@ -89,15 +89,15 @@ const TopicMemberScreen = () => {
     const result = await getAllMemberTopic(query);
     if (result.code === 200) {
       const newDataFollowed = result.data
-        .filter((item) => item.is_following && profile.myProfile.user_id !== item.user_id)
+        .filter((item) => item.is_following)
         .map((data) => ({
           ...data,
           name: data.username,
           image: data.profile_pic_path,
           description: null
         }));
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here are logical and improve the code's readability. The removal of the condition `profile.myProfile.user_id !== item.user_id` simplifies the filter operation, making it easier to understand. However, please ensure that this change does not introduce any unintended side effects, such as including the current user in the list of followed or unfollowed users.

```diff
- .filter((item) => item.is_following && profile.myProfile.user_id !== item.user_id)
+ .filter((item) => item.is_following)
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->